featurebyte/models/online_store.py

```diff
@@ -127,3 +135,113 @@ def value_type(self) -> str:
         """
         adapter = get_sql_adapter(self.feature.feature_store_type)
         return adapter.get_physical_type_from_dtype(self.feature.dtype)
+
+
+class RedisOnlineStoreDetails(FeatureByteBaseModel):
+    """
+    Configuration details for Redis online store.
+
+    Examples
+    --------
+    >>> details = fb.RedisOnlineStoreDetails(
+    ...   redis_type="redis",
+    ...   connection_string="localhost:6379",
+    ...   key_ttl_seconds=3600
+    ... )
+    """
+
+    __fbautodoc__ = FBAutoDoc(proxy_class="featurebyte.RedisOnlineStoreDetails")
+
+    # Online store type selector
+    type: Literal[OnlineStoreType.REDIS] = OnlineStoreType.REDIS
+
+    redis_type: RedisType = Field(
+        default=RedisType.REDIS, description="Redis type: redis or redis_cluster."
+    )
+    #  format: host:port,parameter1,parameter2 eg. redis:6379,db=0
+    connection_string: StrictStr = Field(
+        default="localhost:6379",
+        description="Connection string with format 'host:port,parameter1,parameter2' eg. redis:6379,db=0",
+    )
+
+    key_ttl_seconds: Optional[int] = Field(
+        default=None, description="Redis key bin ttl (in seconds) for expiring entities."
+    )
+
+
+class MySQLOnlineStoreDetails(FeatureByteBaseModel):
+    """
+    Configuration details for MySQL online store.
+
+    Examples
+    --------
+    >>> details = fb.MySQLOnlineStoreDetails(
+    ...   host="localhost",
+    ...   user="user",
+    ...   password="password",
+    ...   database="database",
+    ...   port=3306
+    ... )
+    """
+
+    __fbautodoc__ = FBAutoDoc(proxy_class="featurebyte.MySQLOnlineStoreDetails")
+
+    # Online store type selector
+    type: Literal[OnlineStoreType.MYSQL] = OnlineStoreType.MYSQL
+
+    host: StrictStr = Field(default="localhost", description="MySQL connection host.")
+
+    user: StrictStr = Field(description="MySQL connection user.")
+
+    password: StrictStr = Field(description="MySQL connection password.")
```

Comment:
Is there a plan for encryption? Could `UsernamePasswordCredential` be used here?

Reply:
We can. For redis the password is part of the connection string - should we store it separately?

Reply:
I think we can store it separately and format the connection string dynamically.